src/domain/platform/search/SearchResultSection.tsx

```diff
@@ -27,11 +28,14 @@ const SearchResultSection: FC<ResultSectionProps> = ({
   loading,
 }) => {
   const titleWithCount = useMemo(() => `${title} (${results.length})`, [title, results.length]);
-
+  const { t } = useTranslation();
+  let resultDisclaimer: string | undefined = undefined;
+  if (results.length >= 8) resultDisclaimer = t('pages.search.results-disclaimer');
```

Comment:
Should be > 8 I think.

Reply:
const resultDisclaimer = results.length > 8 ? t('pages.search.results-disclaimer') : undefined